frontend/packages/console-shared/src/utils/alert-utils.ts

```diff
@@ -7,9 +7,17 @@ export const sortMonitoringAlerts = (alerts: Alert[]): Alert[] =>
 
 export const getSeverityAlertType = (alerts: Alert[]): AlertSeverity => {
   const sortedAlerts = sortMonitoringAlerts(alerts);
-  const severityType = (sortedAlerts[0]?.labels?.severity as AlertSeverity) ?? null;
+  const severityType = (sortedAlerts[0]?.labels?.severity as AlertSeverity) ?? AlertSeverity.None;
   return severityType;
 };
 
 export const getFiringAlerts = (alerts: Alert[]): Alert[] =>
   _.filter(alerts, (alert) => alert.state === AlertStates.Firing);
+
+export const shouldHideMonitoringAlertDecorator = (
+  firingAlerts: Alert[],
+  severityAlertType: AlertSeverity,
+): boolean =>
+  firingAlerts.length === 0 ||
```

Comment:
If the `getSeverityAlertType` method were to return `AlertSeverity.None` when no alerts were passed (rather than null) this could be removed and the alerts would not need to be passed.